test/Makefile

```diff
@@ -20,11 +20,10 @@ GO_VALIDATION_ERR = Go version is not supported. Please update to at least $(MIN
 
 all: $(TEST_BINARIES) phaul-test
 	mkdir -p image
-	PID=$$(piggie/piggie) && { \
+	PID=$$(piggie/piggie) && \
+	trap "kill -9 $$PID" INT EXIT TERM && \
```

Comment:
```Suggestion
	trap "pkill -9 piggie" INT EXIT TERM && \
```
When running `make test` it shows the following:

> /bin/bash: line 1: kill: (22507) - No such process

Reply:
Right. piggie creates a PID namespace and therefore the PID changes after restore. The original code using pkill is probably needed.

Reply:
Agh I forgot to run the tests after amending the commit. Fixing this ASAP